src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -387,6 +393,83 @@ def set_obspace_attributes(VarAttrs):
     return VarAttrs
 
 
+def remapBG(input_files):
+
+    print('Remap ATMS: Backus–Gilbert Inversion method')
+
+    obs_data = init_obs_loc()
+
+    src_ch = [1, 2]
+    tgt_ch = [1, 2]
+    src_bm = 5.2
+    tgt_bm = 3.3
+    nfov = 96
+
+    orb = 'des'
+    orbit_num = 'non'
+    ta_vmin, ta_vmax = 210, 275
+    dif_vmin, dif_vmax = -5, 5
+
+    coef_dir = './testinput/'
+    apply_BG_obj = apply_BG_class(input_files,
+                                  orb, orbit_num,
+                                  src_ch,
+                                  coef_dir,
+                                  nfov,
+                                  ta_vmin, ta_vmax,
+                                  dif_vmin, dif_vmax)
+    apply_BG_obj.ingest()
+    apply_BG_obj.prepcoef()
+
+    lat = []
+    lon = []
+    ta = []
+    ta_rmp = []
+    satzen = []
+    satazi = []
+    solzen = []
+    solazi = []
+    tim = []
+    taAllCh = []
+    viewang = []
+    [viewang, taAllCh, lat, lon, satzen, satazi, solzen, solazi, tim] = apply_BG_obj.apply()
+
+    g = h5py.File(input_files[0], 'r')
```

Comment:
think this line can be removed?

Reply:
> think this line can be removed?

In line 444, "g" is needed for "sensorChannelNumber".